sbom/internal/formats/syft2/encoder_test.go

```diff
@@ -23,8 +23,8 @@ func TestDirectoryEncoder(t *testing.T) {
 	testutils.AssertEncoderAgainstGoldenSnapshot(t,
 		Format(),
 		testutils.DirectoryInput(t),
-		true,
 		*updateJson,
+		testutils.TypeJson,
```

Comment:
> Can you point to the tests that mixed up the order? I didn't see any changes like that in this diff?

E.g. here

you can see it was `true, *updateJson` now it is `*updateJson, testutils.TypeJson`

This caused the test to:
- compare the file not as a json unless you provide the `--update-json` flag, which is supposed to update the golden files and default to false
- always update the golden files

There are multiple similar cases.

> Unless you mean the tests are in other buildpacks, not packit?

This should not be the case, as it is in the `internal` sub-package (otherwise this would be a breaking change).

Reply:
Oh, yeah, of course. Thanks! It was so subtle that I missed it despite actively looking for it.